socorro/unittest/external/elasticsearch/test_supersearch.py

```diff
@@ -1533,6 +1533,48 @@ def test_get_with_pagination(self):
         eq_(res['total'], 21)
         eq_(len(res['hits']), 0)
 
+    def test_get_with_sorting(self):
```

Comment:
Locally it passes with ES 1.4. Incidentally, I realized that tests in ``socorro.unittest.external.elasticsearch.test_index_cleaner.IntegrationTestIndexCleaner`` do not... I'll file a bug for that, it's slightly out of the scope here.